requirements.txt

```diff
@@ -1,9 +1,12 @@
 black==22.3.0
-django==3.1.3
-flake8==3.8.4
+django==3.2
```

Comment:
Can we update it to django 4.x series?

Reply:
Yes I can do that, but I have to update the python version to 3.10.5, is that ok? 
it would be interesting if the python version was explained, I thought it would be .python-version but it's in .gitignore.